notebooks-cpd/Event_Store_Data_Feed.ipynb

```diff
@@ -0,0 +1,276 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data feed (optional)\n",
+    "\n",
+    "Using a notebook to feed data into Event Store is now optional for this code pattern.\n",
+    "Instead, you should use IBM Streams to feed the data. If you are not using IBM Streams,\n",
+    "you can run this notebook as an alternative.\n",
+    "\n",
+    "In this notebook, a CSV file is read into a pandas DataFrame\n",
+    "which is then used to batch insert the rows into Event Store.\n",
+    "\n",
+    "Before running this notebook, you need to run the **Event Store Table Creation**\n",
+    "notebook to setup the database and table."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to IBM Db2 Event Store\n",
+    "\n",
+    "Edit the values in the next cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CONNECTION_ENDPOINT=\"\"\n",
+    "\n",
+    "EVENT_USER_ID=\"\"\n",
+    "\n",
+    "EVENT_PASSWORD=\"\"\n",
+    "\n",
+    "# Port will be 1100 for version 1.1.2 or later (5555 for version 1.1.1)\n",
+    "PORT = \"30370\"\n",
+    "\n",
+    "DEPLOYMENT_ID=\"\"\n",
+    "\n",
+    "# Database name\n",
+    "DB_NAME = \"EVENTDB\"\n",
+    "\n",
+    "# Table name\n",
+    "TABLE_NAME = \"IOT_TEMPERATURE\"\n",
+    "\n",
+    "HOSTNAME=\"\"\n",
+    "\n",
+    "DEPLOYMENT_SPACE=\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "bearerToken=!echo `curl --silent -k -X GET https://{HOSTNAME}:443/v1/preauth/validateAuth -u admin:password |python -c \"import sys, json; print(json.load(sys.stdin)['accessToken'])\"`\n",
+    "bearerToken=bearerToken[0]\n",
+    "keystorePassword=!echo `curl -k --silent  GET -H \"authorization: Bearer {bearerToken}\" \"https://{HOSTNAME}:443/icp4data-databases/{DEPLOYMENT_ID}/zen/com/ibm/event/api/v1/oltp/keystore_password\"`\n",
+    "keystorePassword"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Python modules\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Note: Only run this cell if your IBM Db2 Event Store is installed with IBM Cloud Pak for Data (CP4D)\n",
+    "\n",
+    "# In IBM Cloud Pak for Data, we need to create link to ensure Event Store Python library is \n",
+    "# properly exposed to the Spark runtime.\n",
+    "import os\n",
+    "src = '/home/spark/user_home/eventstore/eventstore'\n",
+    "dst = '/home/spark/shared/user-libs/python3.6/eventstore'\n",
+    "try:\n",
+    "    os.remove(dst)\n",
```

Comment:
same remove.  Be super careful not to remove other peoples stuff.  No need here.  Just create, maybe validate.